src/components/WebexActivityStream/WebexActivityStream.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import RoomsJSONAdapter from '../../adapters/RoomsJSONAdapter';
+import {RoomType} from '../../adapters/RoomsAdapter';
+import rooms from '../../data/rooms';
+
+import WebexActivityStream from './WebexActivityStream';
+
+jest.mock('../hooks/useRoom');
+jest.mock('../hooks/useActivityStream');
+
+describe('Webex Avatar component', () => {
+  let roomID, newRooms, roomsAdapter;
+
+  beforeEach(() => {
+    [roomID] = Object.keys(rooms);
+    newRooms = rooms;
```

Comment:
What does this do?

Reply:
resets it back to default

Reply:
Why don't you move lines 49-52 to here? That's basically resetting it too and it's not in two places (e.g. before test and in the test)